@xen-orchestra/web-core/lib/components/console/VtsLayoutConsole.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div :class="uiStore.isMobile ? 'mobile' : undefined" class="vts-layout-console">
+    <slot />
+    <UiCard>
+      <slot name="actions" />
+    </UiCard>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import UiCard from '@core/components/ui/card/UiCard.vue'
+import { useUiStore } from '@core/stores/ui.store'
+
+defineSlots<{
+  default(): any
+  actions(): any
+}>()
+
+const uiStore = useUiStore()
+</script>
+
+<style lang="postcss" scoped>
+.vts-layout-console {
+  display: flex;
+  gap: 2.4rem;
+  padding: 0.8rem;
+
+  &.mobile {
+    flex-direction: column;
+  }
+}
+.ui-card {
+  height: fit-content;
+  gap: 1.6rem;
+  padding: 1.6rem;
+  width: 43rem;
+}
```

Comment:
To keep consistency with the other components in `web-core`, I think we can nest this rule inside `.vts-layout-console`.
Also, to add more readability, you can add a class to the `UiCard` component and use this class here. And in case the root class of `UiCard` changes, there would be no need for updating the selector here.
The same applies in `VtsConsoleActions` component, on the `.ui-button` class.
```html
<template>
  <UiCard class="card" />
</template>

<style>
.card { /* ... */ }
</style>
```

Reply:
Remember to apply the same change in `VtsConsoleActions` component, on the `.ui-button` class.